### ISSUE TITLE
assembly bias test sims

### ISSUE THREAD

**Issue by changhoonhahn** [changhoonhahn/simbig#26]
As one of the tests to validate our SBI framework, we will demonstrate that we can derive unbiased cosmological constraints from forward modeled sims with assembly bias.

We will need to

[ ] construct assembly bias test sims
[ ] measure summary statistics of assembly bias test sims
[ ] derive posteriors for assembly bias test sims
[ ] validate the NPE

**Comment by changhoonhahn** [comment]
@emassara would it be possible for you to check whether assembly bias will be necessary for the marked power spectrum? 

I think a 0th order check would be to measure M(k) for the training mocks and then assembly bias mocks and comparing the general shape of them.

**Comment by emassara** [comment]
@changhoonhahn I was actually planning on performing this test up to the inference part, I can do the 0th order first. Do we already have the mocks with assembly bias?

**Comment by changhoonhahn** [comment]
Yes. I constructed assembly bias test sims in c60cfae6928a0c9b8d20de3531aa3e38c06c48fe

You can find them on tiger in the directory: `/tigress/chhahn/simbig/cmass/tests/`
They're in the format: `hod.quijote_fid_hr*.z0p5.cmass_sgc.abias*.bf`
There are 10 fiducial cosmologies x 5 fiducial HOD + assembly bias realizations
